src/core/mac/sub_mac.cpp

```diff
@@ -396,12 +396,13 @@ void SubMac::StartCsmaBackoff(void)
         {
             static constexpr uint32_t kAheadTime = kCcaSampleInterval + kCslTransmitTimeAhead + kRadioHeaderShrDuration;
             Time                      txStartTime = Time(mTransmitFrame.mInfo.mTxInfo.mTxDelayBaseTime);
+            Time                      now         = Time(static_cast<uint32_t>(otPlatRadioGetNow(&GetInstance())));
```

Comment:
Thanks for adding a variable for this. How about calling `radioNow` to make it clear it is the radio time reference.